### DISCUSSION TITLE
Module not found: Error: Can't resolve 'iobuffer' in '**/node_modules/fast-png/lib-esm'

### DISCUSSION THREAD

**Discussion by AnthonyHao** [pnpm/pnpm#7398] [Q&A] [answered]
After I removed `shamefully-hoist=true` in `.npmrc`, encountered this error. I knew that `fast-png` is dependent in the `package.json`, and the `iobuffer` is only dependent in the `fast-png`. When I remove `shamefully-hoist`, the program can not find the modules. `iobuffer` exist in `node_nodules/.pnpm/node_modules`.

 I tried to add `resolve.fallback: { iobuffer: require. resolve('iobuffer')}` or set `resolve.modules: [node_nodules/.pnpm/node_modules]` in webpack config. It seems work.

And I don't want to add the module in `public-hoist-pattern`.

Is there another way for webpack config?  Thanks.

**Comment by thynson** [comment, marked as answer]
You can use [packageExtentions](https://pnpm.io/package_json#pnpmpackageextensions) to fix the missing peer declaration of it.
